CMakeLists.txt

```diff
@@ -470,7 +470,7 @@ function(is_sdk_requested name result_var_name)
     set("${result_var_name}" "TRUE" PARENT_SCOPE)
   else()
     if("${SWIFT_SDKS}" STREQUAL "")
-      set("${result_var_name}" "TRUE" PARENT_SCOPE)
+      set("${result_var_name}" "FALSE" PARENT_SCOPE)
     else()
```

Comment:
Can we just drop the `if` part now?  If the list is empty, the item is not in the list, and the `else` branch will return false.